wgpu-hal/src/metal/command.rs

```diff
@@ -348,15 +366,15 @@ impl crate::CommandEncoder for super::CommandEncoder {
                 // the amount of data to copy.
                 0
             };
-            encoder.copy_from_buffer_to_texture(
+            encoder.copyFromBuffer_sourceOffset_sourceBytesPerRow_sourceBytesPerImage_sourceSize_toTexture_destinationSlice_destinationLevel_destinationOrigin_options(
```

Comment:
thought: Oh wow, this (what I presume to be an) overload name is _awful_. 😆

Reply:
Yeah, if you have suggestions for better ways of doing this naming, feel free to drop them in https://github.com/madsmtm/objc2/issues/284.